Automattic/wp-calypso
# Hosting Panel: Stub out UI

### Summary

Stubbing out the UI section for the Atomic hosting panel. Also adding some Material Icon assets.

Some things to be addressed in a future PR:
- Routing still needs to be reliant on an endpoint for conditionally allowing a user to navigate to the panel.
- Dummy info needs to be replaced with info from the API endpoint.

<hr>

### Screenshots

<img width="1167" alt="Screen Shot 2019-10-06 at 11 25 50 PM" src="https://user-images.githubusercontent.com/8892849/66283721-95f15080-e892-11e9-8fe7-4dd0e6bee88d.png">

<hr>

### Testing

- Spin up a local Calypso dev environment.
- Go to `/hosting`.
- Select an Atomic site. Make sure that you see the new cards.

Fixes #36280
